lib/db/mem.js

```diff
@@ -271,6 +287,7 @@ module.exports = function (log, error) {
             throw error.notFound()
           }
           var device = account.devices[deviceKey]
+          // If changing sessionTokenId, the old token loses its device record.
           if (device.sessionTokenId) {
             if (deviceInfo.sessionTokenId) {
```

Comment:
What if a device HAD a sessionTokenId and no longer does because a refreshTokenId is passed in instead? Worth worrying about?